src/PlaywightSharp.ProtocolTypesGenerator/Chromium/ChromiumProtocolTypesGenerator.cs

```diff
@@ -99,13 +99,13 @@ private void PrepareArrayTypes(ChromiumProtocolDomainsContainer response)
                     }
                     else if (type.Type == "integer")
                     {
-                        _knownTypes[type.Id] = "int";
-                        _knownTypes[$"{domain.Domain}.{type.Id}"] = "int";
+                        _knownTypes[type.Id] = "int?";
+                        _knownTypes[$"{domain.Domain}.{type.Id}"] = "int?";

```

Comment:
Properties need to be nullable, if we don't do this we would send properties with primitives default values, such as 0 for timeout